File: sdkwork-app-sdk-typescript/src/types/batch-image-audit-form.ts
import type { ImageItem } from './image-item';

/** 批量图片审核请求 */
export interface BatchImageAuditForm {
  /** 图片列表 */
  images: ImageItem[];
  /** 检测类型 */
  checkTypes?: string[];
  /** 场景类型 */
  scene?: string;
}

File: sdkwork-app-sdk-java/src/main/java/com/sdkwork/app/model/BatchImageAuditForm.java
package com.sdkwork.app.model;

public class BatchImageAuditForm {
    private List<ImageItem> images;
    private List<String> checkTypes;
    private String scene;

    public List<ImageItem> getImages() {
        return this.images;
    }
    
    public void setImages(List<ImageItem> images) {
        this.images = images;
    }

    public List<String> getCheckTypes() {
        return this.checkTypes;
    }
    
    public void setCheckTypes(List<String> checkTypes) {
        this.checkTypes = checkTypes;
    }

    public String getScene() {
        return this.scene;
    }
    
    public void setScene(String scene) {
        this.scene = scene;
    }
}
